src/storage/src/hummock/sstable_store.rs

```diff
@@ -105,7 +105,8 @@ pub enum CachePolicy {
     /// Disable read cache and not fill the cache afterwards.
     Disable,
     /// Try reading the cache and fill the cache afterwards.
-    Fill,
+    ///  true means it will fill a high priority block.
+    Fill(bool),
```

Comment:
Due to the ambiguity between Fill(false) and NotFill, how about using a new enum `Priority(High, Low)` to replace `Fill(bool)` with `Fill(Priority)`?

Reply:
Good idea.